src/main/java/uk/gov/hmcts/reform/draftstore/endpoint/v2/DraftStoreEndpoint.java

```diff
@@ -27,6 +28,12 @@
 import static org.springframework.web.bind.annotation.RequestMethod.POST;
 import static uk.gov.hmcts.reform.draftstore.domain.SaveStatus.Updated;
 
+@APIDeprecated(
+    name = "/api/v2/draft endpoint",
+    expiryDate = "TBD",
+    docLink = "about:blank",
+    note = "Use new `/drafts` endpoint instead."
+)
```

Comment:
Can we make expiryDate and docLink optional? If now, are those values ok?

Reply:
What is the expiry date? i.e. when the api will be removed?

Reply:
well, we don't know

Reply:
would probably be best to leave it blank if it's not defined and have it conform to ISO 8601 format so that it can be processed by machine